src/main/java/uk/ac/ebi/eva/evaseqcol/service/SeqColService.java

```diff
@@ -446,20 +451,20 @@ public List<String> getCommonElementsDistinct(List<String> seqColAFields, List<S
     /**
      * Return the number of common elements between listA and listB
      * Note: Time complexity for this method is about O(n²)*/
-    public Integer getCommonElementsCount(List<String> listA, List<String> listB) {
-        List<String> listALocal = new ArrayList<>(listA); // we shouldn't be making changes on the actual lists
-        List<String> listBLocal = new ArrayList<>(listB);
+    public Integer getCommonElementsCount(List<?> listA, List<?> listB) {
+        List<?> listALocal = new ArrayList<>(listA); // we shouldn't be making changes on the actual lists
```

Comment:
Can't we convert these to sets and do a set intersection?

Reply:
Yep, I assume so.
I'll do that.